week-2/13-order/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Flexbox</title>
```

Comment:
Also, I've been using the exercise number in the title, because I think students often have so many tabs open and struggle to find what they're working on. Something like:

> 13 - Flex Order - HTML, CSS and Git Exercises

And same for 14 below.